test/test_mps.py

```diff
@@ -6140,40 +6140,6 @@ def test_byte_mask_accumulate(self, device="mps"):
             self.assertEqual(y, torch.ones(size=(10, 10), device=device))
             self.assertEqual(len(w), 2)
 
-    def test_index_put_accumulate(self, device="mps"):
```

Comment:
Why did we remove this test ?

Reply:
The conversion logic from test_indexing.py to test_mps.py was wrong - this was testing initially large tensors, which are not supported on MPS:
**test.indexing.py**:
```
    def test_index_put_accumulate_large_tensor(self, device):
        # This test is for tensors with number of elements >= INT_MAX (2^31 - 1).
        N = (1 << 31) + 5
```
I quickly converted it to use a smaller size, but this proved to lead to correctness issues and other failures.